conans/model/ref.py

```diff
@@ -19,10 +19,10 @@ def validate_conan_name(name):
                           " at least %s characters." % (name, ConanFileReference.min_chars)
             else:
                 message = "'%s' is an invalid name. Valid names MUST begin with a "\
-                            "letter or number, have between %s-%s chars, including "\
```

Comment:
I tend to do that too, except I normally go by commit rather than full PR (assuming the intention of the PR is changing functionality) otherwise you'd have to chain multiple PRs. That said, each repo normally has a different workflow so I can follow that one as well